lib/rspec/sidekiq/matchers/have_enqueued_sidekiq_job.rb

```diff
@@ -1,11 +1,11 @@
 module RSpec
   module Sidekiq
     module Matchers
-      def have_enqueued_job(*expected_arguments)
-        HaveEnqueuedJob.new expected_arguments
+      def have_enqueued_sidekiq_job(*expected_arguments)
+        HaveEnqueuedSidekiqJob.new expected_arguments
       end
```

Comment:
Personally I don't think having that alias is a good idea. It's better to make people upgrade the gem now if it's not working for them (because of rspec-rails), or if they would need new features in the future. If it's a major version bump it should be at least semi-obvious that there is a breaking change.
I think the alias will cause problems, people might complain about ArgumentErrors or sporadic issues which will be caused by gem load order or similar, it will be difficult to pinpoint where the issue is. If we just rename the method there is no problem. Also, most developers know how to search and replace in all project files. So to fix the user specs is a matter of seconds.

I'm not saying `have_enqueued_sidekiq_job` is necessarily the best name but I am saying pick something and stick with that. Having aliases will just confuse everyone, especially if they aren't aware of what rspec-rails provides.